aws/aws.go

```diff
@@ -796,7 +808,18 @@ func (s *awsOps) Create(
 		return nil, cloudops.NewStorageError(cloudops.ErrVolInval,
 			"Drive type not specified in the storage spec", "")
 	}
-
+	if vol.AvailabilityZone == nil {
```

Comment:
According to CreateVolumeInput documentation
- AvailabilityZone is a required field
- The size of the volume, in GiBs. You must specify either a snapshot ID or a volume size.
- The snapshot from which to create the volume. You must specify either a snapshot ID or a volume size.